implementation/src/main/java/io/smallrye/mutiny/groups/UniCreate.java

```diff
@@ -256,6 +256,15 @@ private UniCreate() {
         return item(() -> item);
     }
 
+    /**
+     * Creates a new {@link Uni} that completes immediately after being subscribed to with a {@code null} item.
+     *
+     * @return the new {@link Uni} with a {@code null} item
+     */
+    public Uni<Void> voidItem() {
+        return item(() -> null);
```

Comment:
that required calling the supplier while a `(Void) null` would be slightly more efficient.

Reply:
we don't really care about producing `null` at subscription time. `null` is `null` regardless of when.

Reply:
I think I'm being dense, do you mean doing `Uni.createFrom().deferred()` like what is used inside `item()`, or something else?

Reply:
Or do you mean creating a `UniVoid` instance and use it in a similar way to `nothing()` method?

Reply:
I believe it should be `item((Void) null))` or `item(NULL_VALUE)` with `NULL_VALUE` a constant set to `null`.

The difference is that the item value is produced when the `Uni` is created and not a subscription time. However, because here the value is constant, it's a bit better. Using `() -> null)` is valid but it means that the `null` value is produced on every subscription and at subscription time. As the value is constant, it does not provide benefits and requires calling the supplier on every subscription. 

Also because this `Uni` is actually _constant_, we may want to write is as follows:

```
public Uni<Void> voidItem() {
        return UNI_OF_VOID;
}

// ...
private static final Uni<Void> UNI_OF_VOID = Uni.createFrom().item((Void) null));
```

Reply:
ah!

Ok, will try the static way, as we don't need many instances of them